tests/testimonialCarousel.test.ts

```diff
@@ -14,21 +14,21 @@ describe('Testimonial - Carousel', () => {
 
         await testimonialActive.dragAndDrop({ x: 100, y: 0 })
         const coryDrag = await $('.swiper-slide-next .slick-slide').$('.elementskit-author-name=Cory Clemmons')
+        
         await browser.pause(2000)       
         await testimonialActive.dragAndDrop({ x: -100, y: 0 })
         await browser.pause(3000)
-
         await expect(coryDrag).toBeExisting()
```

Comment:
Please fix the variables naming, the variables names must be the most describable possible being reference to the object that variable contains, for example, const searchButton = $('...')...

variables as eric, stephen don't tell us anything about what is contained, for example, const ericSlideCard is more describable about what is inside of that variable, please use this logic to rename your variables